README.md

```diff
@@ -1 +1,276 @@
+recommendations to customers who have bought a particular item.
+
+![graph-image](/images/graph-image.png)
+
+Customers Dino and Durga bought the same items. We are able to traverse the
+graph and recommend items to Dallas based on the items that Dino and
+Durga bought. This is a simple example of how graph databases can be used to
+offer recommendations to customers.
+
+## ❓ How to use this demo
+
+You can use this demo to understand how to use Macrometa SDK to create graph
+database and perform operations on it. In the sample data folder, you will find
+sample data that you can use to import in Macrometa GDN. We suggest that you
+follow the steps explained in [Graph data setup](#graph-data-setup) section to import data and
+create a graph. After that,  you can interact with the graph using the scripts in `create`,
+`read`, `update`, and `delete` folders. (CRUD operations on graph)
+
+## 🚀 Run It Locally
+In this tutorial, we will walk you through the process of setting up a local instance of a Macrometa graph example project. By following this step-by-step guide, you will learn how to create an account, set up your environment, import sample data and queries, and interact with the Macrometa Global Data Network (GDN) using JavaScript SDK. Additionally, we will demonstrate how to add new vertices and edges to the graph. Let's get started by setting up the prerequisites and initial environment.
+
+### Prerequisites
+
+- Log in to your [Macrometa account](https://auth-play.macrometa.io/)
+- You need **Node.js 14 (or later)** installed in your system. Run
+   the following command in your CLI to check your Node.js version.
+
+```
+node -v
+```
+
+### Initial Setup
+
+In this paragraph, we will explain how to clone this repository and set up environment variables.
+
+1. Open a terminal window and run the command below to clone this repository. You can run:
+
+```
+git clone https://github.com/Macrometacorp/macrometa-graph-example.git
+```
+
+or
+
+```
+git clone git@github.com:Macrometacorp/macrometa-graph-example.git
+```
+
+depending on your GitHub authentication method.
+
+2. Create a new `.env` file and add your environment variables on your selected
+   path.  
+   Here is how a `.env` file should look:
+
+```
+BASE_URL=https://play.paas.macrometa.io/
+API_KEY=<your-api-key>
+FABRIC=<your-fabric-name> (default: _system)
+```
+
+**Note**: `.env.sample` is only a sample file that you can also use as a guide to create your own `.env`.
+
+3. Install the project dependencies with:
+
+```
+npm install
+```
+
+### Graph Data Setup
+
+1. Create two document collections in Macrometa GDN using Macrometa console.
+
+   - `groceryItems`
+   - `groceryCustomers`
+
+2. Create one edge collection in Macrometa GDN using Macrometa console.
+
+   - `groceryOrders`
+
+3. Import test data from `sample-data` folder in Macrometa GDN.
+
+4. Import `sample-data/import-query.json` in Macrometa GDN using Macrometa
```

Comment:
For instructions, refer to [Import Query Workers](https://www.macrometa.com/docs/queryworkers/query-workers#import-query-workers).